src/test_workflow/benchmark_test/benchmark_args.py

```diff
@@ -0,0 +1,111 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import argparse
+import json
+import logging
+from typing import IO
+
+from test_workflow.test_jsonargs import JsonArgs
+
+
+# Contains the arguments required to run a perf test.
+class BenchmarkArgs:
+    bundle_manifest: IO
+    stack_suffix: str
+    config: IO
+    keep: bool
+    insecure: bool
+    singleNode: bool
+    minDistribution: bool
+    managerNodeCount: int
+    dataNodeCount: int
+    clientNodeCount: int
+    ingestNodeCount: int
+    mlNodeCount: int
+    dataNodeStorage: int
+    mlNodeStorage: int
+    jvmSysProps: str
+    additionalConfig: str
+    workload: str
+    benchmarkConfig: IO
+    userTag: str
+    targetHosts: str
+    logging_level: int
+
+    def __init__(self) -> None:
+        parser = argparse.ArgumentParser(description="Test an OpenSearch Bundle")
+        parser.add_argument("--bundle-manifest", type=argparse.FileType("r"), help="Bundle Manifest file.",
+                            required=True)
+        parser.add_argument("--suffix", dest="suffix", help="Suffix to be added to stack name for performance test")
+        parser.add_argument("--component", dest="component", default="OpenSearch",
+                            help="Component name that needs to be performance tested")
+        parser.add_argument("--config", type=argparse.FileType("r"), help="Config file.", required=True)
+        parser.add_argument(
+            "--without-security", dest="insecure", action="store_true",
+            help="Force the security of the cluster to be disabled.", default=False)
+        parser.add_argument("--keep", dest="keep", action="store_true",
+                            help="Do not delete the working temporary directory.")
+        parser.add_argument("--singleNode", dest="singleNode", action="store_true",
```

Comment:
Are arguments elsewhere `camelCase`? I think not.

Reply:
this was a miss, updated to use uniform formatting.